web-app/uni-todo-app/src/components/TodoInputField.tsx

```diff
@@ -9,21 +9,29 @@ export const TodoInputField = () => {
   const dispatch = useAppDispatch();
 
   const addTodoEntity = useCallback(() => {
-    const todoEntity: ITodoEntity = {
-      name: input,
-      done: false,
-      createdAt: new Date().toISOString(),
-      updatedAt: new Date().toISOString(),
-    };
-    dispatch(insertTodoEntity(todoEntity));
-    setInput("");
+    if (input.trim() == "") alert("TODO field cannot be empty")
```

Comment:
in general it is better to use === in TypeScript. Which will also check for the type of the variable.
In this case `if (!!input.trim())` will also do the trick

Reply:
An early return is also more beneficial and readable.

```
if (!!input.trim()) {
// I would also replace the alert with something else (https://v1.tailwindcss.com/components/alerts)
 return;
}
const todoEntity: ITodoEntity = {
        name: input,
        done: false,
        createdAt: new Date().toISOString(),
        updatedAt: new Date().toISOString(),
};
dispatch(insertTodoEntity(todoEntity));
setInput("");

```